ignite/templates/module/module.go

```diff
@@ -9,8 +9,8 @@ import (
 
 // ProtoPackageName creates a protocol buffer package name for an app module.
 func ProtoPackageName(appModulePath, moduleName string) string {
-	path := strings.Split(appModulePath, "/")
-	path = append(path, moduleName)
+	pathArray := strings.Split(appModulePath, "/")
+	path := []string{pathArray[len(pathArray)-1], moduleName}
```

Comment:
Why is only the last directory used instead of the full relative path?

Reply:
the path doesn't need the namespace anymore